cometd-java/cometd-java-benchmark/cometd-java-benchmark-server/src/main/java/org/cometd/benchmark/server/CometDLoadServer.java

```diff
@@ -541,16 +548,20 @@ public long convert(long measure) {
         }
 
         private void reset() {
-            histogram.reset();
+            allHistograms.forEach(AbstractHistogram::reset);
```

Comment:
Since we are using `Histogram` everywhere, replace `AbstractHistogram::reset` with just `Histogram::reset`.

Reply:
Ok.